shell/browser/api/electron_api_safe_storage.cc

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021 Slack Technologies, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/api/electron_api_safe_storage.h"
+
+#include <vector>
+
+#include "components/os_crypt/os_crypt.h"
+#include "shell/browser/browser.h"
+#include "shell/common/gin_converters/base_converter.h"
+#include "shell/common/gin_converters/callback_converter.h"
+#include "shell/common/gin_helper/dictionary.h"
+#include "shell/common/gin_helper/object_template_builder.h"
+#include "shell/common/node_includes.h"
+#include "shell/common/platform_util.h"
+
+namespace electron {
+
+namespace api {
+
+#if DCHECK_IS_ON()
+bool SafeStorage::electron_crypto_ready = false;
+#endif
+
+static const char* const kEncryptionVersionPrefix[] = {"v10", "v11"};
```

Comment:
Can we have these as two definitions instead of an array? Would improve readability further down IMO